src/app/views/templates/submit-form-email.server.view.html

```diff
@@ -4,7 +4,12 @@
   <body>
     <p>Hello there,</p>
     <p>A "<%= formTitle %>" form was submitted on <%= submissionTime %>.</p>
-    <br /><br /><br />
+    <% if (responseMode === 'encrypt') { %>
+      <p>View this submission and download attachments on <a href="http://localhost:3000/admin/form/<%= formId %>/results/<%= refNo %>">FormSG</a></p>
```

Comment:
Need to be careful of side effects when linking an authenticated url. We might need to exclude certain url groups from DD to avoid getting false positive alerts / betteruptime calls.